src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -36,6 +36,9 @@ final class CartContext implements Context
     /** @var ApiClientInterface */
     private $productsClient;
 
+    /** @var ApiClientInterface */
+    private $orderClient;
```

Comment:
```suggestion
    private $ordersClient;
```
to be consistent with the rest  of services